### ISSUE TITLE
ptrace_debug_regs is broken on 5.8

### ISSUE THREAD

**Issue by khuey** [mozilla/rr#2636]
```
FAILED: errno=0 (Success)
ptrace_debug_regs: /home/khuey/dev/rr/src/test/x86/ptrace_debug_regs.c:55: main: Assertion `"FAILED: !" && check_cond(status == ((5 << 8) | 0x7f))' failed.
```

bisected to https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/commit/?h=v5.8&id=2bbc68f8373c0631ebf137f376fbea00e8086be7

`status=0x4d00`

**Comment by khuey** [comment]
https://lkml.org/lkml/2020/8/19/966

**Comment by khuey** [comment]
Should be fixed in the kernel now.

**Comment by bernhardu** [comment]
@khuey
I guess these are the fixes in [linux-5.8](https://git.kernel.org/pub/scm/linux/kernel/git/stable/linux.git/commit/arch/x86?h=linux-5.8.y&id=4e99722219f4341fd4ff6f130d4391b2d2d8e0f4) and [linux-5.9](https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/commit/arch/x86?id=d5c678aed5eddb944b8e7ce451b107b39245962d) and therefore fixed in kernels v5.8.8 and v5.9-rc4 and above?

**Comment by khuey** [comment]
I didn't track exactly which releases the fix made it into but I believe both the latest 5.9 rc and the latest 5.8 stable release have the fix.
